doc/source/prometheus.rst

```diff
@@ -0,0 +1,117 @@
+Ansible metrics with Prometheus
+===============================
+
+ara doesn't provide monitoring or alerting out of the box (they are out of scope) but it records a number of granular metrics about Ansible playbooks, tasks and hosts, amongst other things.
+
+Starting with version 1.6.2, ara provides an integration of `prometheus_client <https://github.com/prometheus/client_python>`_ that queries the ara API and then exposes these metrics for prometheus to scrape.
```

Comment:
1.6.2 didn't pan out, we went straight to 1.7.0. It can be included in a release as soon as it's ready.